src/domain/entities/noteList.ts

```diff
@@ -0,0 +1,3 @@
+import type { Note } from '@domain/entities/note';
+
+export type NoteList = Note[];
```

Comment:
I'd suggest wrapping notes array to some property, because NoteList could have another properties as well:

```suggestion
export type NoteList = {
  items: Note[];
}
```

Reply:
I'm not entirely sure that in the future it will be necessary to add some properties, like to an object. The task is to get a list of notes. I don't think there is any need to add any properties to the list

Reply:
We can add properties, because note list is complex feature

Reply:
Changed it as suggested. NoteList is an object.